packages/credentials-interface/package.json

```diff
@@ -47,7 +47,9 @@
     "url": "git+https://github.com/energywebfoundation/ew-did-registry.git"
   },
   "dependencies": {
-    "@sphereon/pex": "1.1.3",
+    "@sphereon/pex": "^2.0.1",
+    "@sphereon/pex-models": "^2.0.2",
```

Comment:
I don't see `@sphereon/pex-models` imported anywhere (though maybe I just missed it 😅 ). Does it need to be a dependency?

Reply:
Yes, we are importing presentation definitions there https://github.com/energywebfoundation/ew-did-registry/blob/e46e2edc7ecd18f14e30b2b10527fff6b737a17f/packages/credentials-interface/src/exchange.utils.ts#L6.

Reply:
Ah okay, thank you